core/Manifest.xml

```diff
@@ -17,6 +17,13 @@
                 <ResourceProvider name = "Microsoft.ContainerService"/>
     </RegisterResourceProviders>
     <ProvisioningSteps>
+        <Manual title="Configure Linux DSVM">
+            <Parameters>
+                <Credential type="linuxvm" username="adminUsername" password="adminPassword" />
+                <Parameter name="vmSize" allowedValues="{VirtualMachineSkus}" defaultValue="Standard_DS3_v2" />
```

Comment:
we have semantic sku selection. otherwise, can run into errors during deployment.